tools/wpt/expectation.json

```diff
@@ -13598,6 +13649,190 @@
       "<a> and <area>.searchParams should be undefined"
     ],
     "idlharness.any.html": true,
+    "toascii.window.html": [
```

Comment:
These tests are never going to pass (they use DOM elements). We should remove them.

Reply:
Some of the tests use the `<a>` and `<area>` elements, but others use the `URL` class.